webpack/assets/javascripts/react_app/components/CommonParameterForm/components/values/BooleanValue.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { translate as __ } from '../../../../common/I18n';
+
+const BooleanValue = ({ id, name, css, value, onChange }) => {
+  const options = [
+    { label: '', value: '' },
+    { label: __('Yes'), value: 'true' },
+    { label: __('No'), value: 'false' },
+  ];
+  return (
+    <select
+      id={id}
+      name={name}
+      className={`without_select2 ${css}`}
+      onChange={e => onChange(e.target.value)}
+      defaultValue={value}
+    >
+      {options.map(item => (
+        <option value={item.value} key={item.value}>
+          {item.label}
+        </option>
+      ))}
+    </select>
```

Comment:
You can use PF4 to get a better look from the html feature :D
https://patternfly-react.surge.sh/components/form-select

something like:
```suggestion
<FormSelect value={defaultValue} onChange={onChange} aria-label="FormSelect Input">
        {options.map((option, index) => (
          <FormSelectOption isDisabled={option.disabled} key={index} value={option.value} label={option.label} />
        ))}
      </FormSelect>
```

Reply:
Looks like PF4 components don't go well with Foreman css:
![pf4](https://user-images.githubusercontent.com/59385976/107951309-6da25780-6f98-11eb-9b5f-5a210f25c6f7.png)

Reply:
Just updating that we decided to try and use the inline form example from PF4: https://patternfly-react.surge.sh/components/form/#horizontal in a react-router page that fetches the `@common_parameter` data